pkg/openstacknet/funcs.go

```diff
@@ -124,11 +122,11 @@ func AddOSNetNameLowerLabels(
 	obj client.Object,
 	cond *ospdirectorv1beta1.Condition,
 	networkNameLowerNames []string,
-) error {
+) map[string]string {
```

Comment:
I think it should return an error as well, as you compare to ``err != nil`` above after the calls being made?